### DISCUSSION TITLE
Update WingetUI

### DISCUSSION THREAD

**Discussion by Breizil29** [microsoft/winget-cli-restsource#206] [General]
Hello everyone,

![WingetUI](https://user-images.githubusercontent.com/81489376/232451914-0e8ace05-5b2f-412b-943b-0889bb61b501.png)

![ccleaner](https://user-images.githubusercontent.com/81489376/232452013-045e12a7-ccfb-41ab-b9f5-e9ce8be86123.png)

**Comment by mdanish-kh** [comment]
https://github.com/marticliment/WingetUI would be the place to file an issue, since WingetUI is external and not maintained by microsoft

**Comment by Breizil29** [comment]
OK
